cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/GetPlatformVmTypesHandler.java

```diff
@@ -38,9 +39,16 @@ public void accept(Event<GetPlatformVmTypesRequest> getPlatformVmTypesRequest) {
             CloudPlatformVariant cloudPlatformVariant = new CloudPlatformVariant(
                     Platform.platform(request.getExtendedCloudCredential().getCloudPlatform()),
                     Variant.variant(request.getVariant()));
-            CloudVmTypes platformVirtualMachinesJson = cloudPlatformConnectors.get(cloudPlatformVariant)
-                    .platformResources()
-                    .virtualMachines(request.getCloudCredential(), Region.region(request.getRegion()), request.getFilters());
+            CloudVmTypes platformVirtualMachinesJson;
+            if (CdpResourceType.DATAHUB.equals(request.getCdpResourceType())) {
+                platformVirtualMachinesJson = cloudPlatformConnectors.get(cloudPlatformVariant)
+                        .platformResources()
```

Comment:
this can be extracted to line 42, since this call is the same in both sides of the if-else block